website/src/pages/en/ai-suite/graph-assistant/faq.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Graph Assistant FAQ
+sidebarTitle: 'FAQ'
+---
+
+This page summarizes some of the most common questions for developers using Graph Assistant.
+
+1. What is The Graph Assistant?
+
+The Graph Assistant is an AI-powered chat interface that lets you query blockchain data in plain English. Under the hood, it translates your questions into GraphQL calls against subgraphs and the Token API, then returns results as a table—no coding required.
+
+2. What is an agentic application?
+
+An agentic application combines AI-driven reasoning with external data sources or APIs to take autonomous actions on your behalf. In this case, Graph Assistant acts as an “agent” that interprets your natural-language prompts, builds and dispatches GraphQL queries, and formats the results so that you don't need to write or manage queries yourself.
```

Comment:
GraphQL and SQL queries. Still an oversimplification, but that's probably fine haha.